webapp/src/components/sidebar/__snapshots__/sidebarBoardItem.test.tsx.snap

```diff
@@ -1,140 +1,63 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`components/sidebarBoardItem sidebar call hideSidebar 1`] = `
+exports[`components/sidebarBoardItem sidebar board item 1`] = `
 <div>
   <div
-    class="SidebarBoardItem"
+    class="SidebarBoardItem subitem active"
   >
     <div
-      class="octo-sidebar-item category ' expanded "
+      class="octo-sidebar-icon"
     >
-      <button
-        class="IconButton"
-        type="button"
-      >
-        <i
-          class="CompassIcon icon-chevron-down ChevronDownIcon"
-        />
-      </button>
-      <div
-        class="octo-sidebar-title category-title"
-        title="Category 1"
-      >
-        Category 1
-      </div>
-      <div
-        aria-label="menuwrapper"
-        class="MenuWrapper"
-        role="button"
-      >
-        <button
-          class="IconButton"
-          type="button"
-        >
-          <i
-            class="CompassIcon icon-dots-horizontal OptionsIcon"
-          />
-        </button>
-      </div>
+      i
```

Comment:
Is this meant to be a tag?

Reply:
I guess it's always been there.  Just looked strange to have a lone `i` outside a tag.

Reply:
No it wasn't, it looks like something that should be part of an "vim" error, probably between the keyboard and the chair, I'll take a look.

Reply:
Ok, this is not an accident, this is because we use unicode emojis, so, this is just a character, and the tests uses the letter `i` for this icon. so... it is working as expected.